packages/build/src/plugins/pinned_version.js

```diff
@@ -108,10 +108,16 @@ const pinPlugin = async function ({
 }) {
   const pinnedVersion = getMajorVersion(version)
   try {
+    const body = {
+      pinned_version: pinnedVersion
+    }
+    if (packageName === '@netlify/plugin-nextjs' && pinnedVersion === 5) {
+      body.pinned_version = 'rc'
```

Comment:
I think we probably just want to not pin it at all in this scenario, because pinning to rc would stop it updating to the stable version. What we could do is check if the `version` is a prerelease, and if so skip pinning entirely

Reply:
You could use `semver`, which we're using elsewhere, and use the `prerelease` function to check

Reply:
Yeah I thought maybe we want to skip pinning it, but I don't know how it ties with the current behaviour. Though now that I think about it, it probably make more sense to not pin it. I'll drop the pinning for now.